src/lib/app-context/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './app-context.functions';
+
+export type { AppContext } from './app-context.model'
```

Comment:
Can we create type declarations for all these models so we can use them in consumers that use typescript? It's pretty easy.